pocketknife-compiler/src/main/java/pocketknife/internal/codegen/TypeUtil.java

```diff
@@ -66,11 +59,6 @@ private TypeUtil(Elements elements, Types types) {
             throw new IllegalStateException("Unable to find Parcelable type");
         }
         parcelableType = element.asType();
-//        element = elements.getTypeElement(BINDER);
```

Comment:
This is technically a todo I forgot to add. I need to handle binders. To do so will require moving to gradle. Removing the semicolons will fix the sonar issue.